corehq/apps/cloudcare/static/cloudcare/js/form_entry/form_ui.js

```diff
@@ -822,6 +861,25 @@ hqDefine("cloudcare/js/form_entry/form_ui", [
         return itemsPerRow !== null ? Math.round(constants.GRID_COLUMNS / itemsPerRow) : constants.GRID_COLUMNS;
     };
 
+    function AddGroup(json, parent) {
```

Comment:
Some of these are to make `AddGroup` act like a `Question` and also like a `Repeat`, right? At least `entryTemplate` and `type` and `newRepeat`? I think it'd be helpful to be explicit about that, probably in a comment.

What would be the pros/cons of this actually inheriting from `Question` or `Repeat`?